modules/nixvim/nixvim.nix

```
@@ -22,6 +22,26 @@ in
       main = lib.mkEnableOption "background transparency for the main NeoVim window";
       signColumn = lib.mkEnableOption "background transparency for the NeoVim sign column";
     };
+    config = lib.mkOption {
+      type = with lib.types; attrsOf anything;
+      readOnly = true;
+
+      description = ''
+        The stylix configuration, generated for nixvim.
+        If nixvim is installed via nixos, darwin, or home-manager then this will be **automatically**
+        assigned to `programs.nixvim`. If you're using a "standalone" build of nixvim, then that's
+        not possible. Instead, you should pass this config to the `nixvimExtend` function.
+        For example:
```

```suggestion
        The stylix configuration, generated for nixvim.

        If nixvim is installed via nixos, darwin, or home-manager then this will be **automatically**
        assigned to `programs.nixvim`. If you're using a "standalone" build of nixvim, then that's
        not possible. Instead, you should pass this config to the `nixvimExtend` function.

        For example:
```

Markdown requires blank lines to signify paragraph breaks.